packages/minecraft.rb

```diff
@@ -3,50 +3,31 @@
 class Minecraft < Package
   description 'A sandbox video game developed by Mojang Studios AB.'
   homepage 'https://minecraft.net'
-  version '2.2.1867'
+  version '2.2.3200'
   license 'Mojang'
   compatibility 'x86_64'
   source_url 'https://launcher.mojang.com/download/Minecraft.deb'
-  source_sha256 '9a8b6b75db199461ae241fbee6adf65fc6636204c58eb79feff968e307464727'
+  source_sha256 '21254b54e59e946adc523898f704592dc7c32b644c618324df80c4b98355ccd0'
 
   depends_on 'libx11'
   depends_on 'jdk8'
-  depends_on 'gtk2'
   depends_on 'gtk3'
+  depends_on 'libcom_err'
   depends_on 'sommelier'
 
   def self.install
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
-    FileUtils.cp_r 'opt', "#{CREW_DEST_PREFIX}"
-    FileUtils.cp_r 'usr/share', "#{CREW_DEST_PREFIX}"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}"
+    FileUtils.cp_r '.', "#{CREW_DEST_PREFIX}/"
+    FileUtils.mv "#{CREW_DEST_PREFIX}/bin/minecraft-launcher", "#{CREW_DEST_PREFIX}/bin/minecraft-launcher.elf"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/.config/minecraft"
     Dir.chdir "#{CREW_DEST_PREFIX}/bin" do
       @launcherscript = <<~EOF
         #!/bin/bash
-        if [[ ! -e /usr/local/.config/minecraft ]]; then
-          mkdir -pv /usr/local/.config/minecraft
-        fi
-        if [[ ! -e $HOME/.var/app/com.mojang.Minecraft ]]; then
-          mkdir -pv $HOME/.var/app/com.mojang.Minecraft
-        fi
-        if [[ -e $HOME/.minecraft ]]; then
-          if [[ ! -L $HOME/.minecraft ]]; then
-            cp -rv $HOME/.minecraft/* /usr/local/.config/minecraft
-            rm -r $HOME/.minecraft
-            ln -sv /usr/local/.config/minecraft $HOME/.minecraft
-          fi
-        else
-          ln -sv /usr/local/.config/minecraft $HOME/.minecraft
-        fi
-        if [[ -e $HOME/.var/app/com.mojang.Minecraft/.minecraft ]]; then
-          if [[ ! -L $HOME/.var/app/com.mojang.Minecraft/.minecraft ]]; then
-            cp -rv $HOME/.var/app/com.mojang.Minecraft/.minecraft/* /usr/local/.config/minecraft
-            rm -r $HOME/.var/app/com.mojang.Minecraft/.minecrft
-            ln -sv /usr/local/.config/minecraft $HOME/.var/app/com.mojang.Minecraft/.minecraft
-          fi
-        else
-          ln -s /usr/local/.config/minecraft $HOME/.var/app/com.mojang.Minecraft/.minecraft
-        fi
-        /usr/local/opt/minecraft-launcher/minecraft-launcher
+        # Minecraft launcher documentation is at:
+        # https://minecraft.fandom.com/wiki/Minecraft_Launcher
+        sudo mount -o remount,symfollow /home/chronos/user
+        #{CREW_PREFIX}/bin/minecraft-launcher.elf --workDir #{CREW_PREFIX}/.config/minecraft --tmpDir #{CREW_PREFIX}/tmp $@
+        sudo mount -o remount,nosymfollow /home/chronos/user
```

Comment:
This is a bit extreme.  Isn't there a more elegant way?

Reply:
FWIW, the `xdg_base` package symlinks `~/.config` to `/usr/local/.config` but that doesn't help with nosymfollow set.

Reply:
cross-device symlinks don't work at all any more, so we should modify  `xdg_base` to stop doing that.